src/main/java/org/mule/tools/apikit/model/Configuration.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.apikit.model;
+
+import java.util.Map;
+
+public class Configuration {
+
+  private String environment;
+  private CommonProperties commonProperties;
```

Comment:
Which is the purpose or advantage of having a CommonProperties and HttpCommonProperty classes? Shouldn't be treated as regular properties. Maybe we can have a "Properties Group" concept to group them semantically, as the configurations.

Reply:
Refactor done.